src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub use tracing_opentelemetry_instrumentation_sdk;
 #[cfg(feature = "otlp")]
 pub mod otlp;
 
-#[cfg(feature = "integration_test")]
+#[cfg(feature = "test")]
```

Comment:
Not sure if I should rename this, but it's only used by profiles right now, so it should be fine